datafusion/core/src/datasource/physical_plan/parquet/statistics.rs

```diff
@@ -126,111 +127,62 @@ macro_rules! get_statistic {
     }};
 }
 
-#[derive(Debug, Clone, Copy)]
-enum Statistic {
-    Min,
-    Max,
-}
-
-/// Extracts statistics for a single leaf column from [`RowGroupMetaData`] as an
-/// arrow [`ArrayRef`]
+/// Lookups up the parquet column by name
 ///
-/// For example, given a parquet file with 3 Row Groups, when asked for
-/// statistics for column "A" it will return a single array with 3 elements.
-pub(crate) struct RowGroupStatisticsConverter<'a> {
-    field: &'a Field,
+/// Returns the parquet column index and the corresponding arrow field
+pub fn parquet_column<'a>(
```

Comment:
This method is basically a hack, if/when we upstream this we can use the parquet-private ParquetField which handles this properly